packages/dscc-gen/src/files.ts

```diff
@@ -113,37 +127,38 @@ export const createAndCopyFiles = async (
     () => createAndCopyFilesImpl(projectPath, templatePath, projectName)
   );
 
-export const remove = async (...directoryParts: string[]): Promise<boolean> => {
+export const remove = (...directoryParts: string[]): boolean => {
   if (directoryParts.length === 0) {
     throw new Error('You must pass directoryParts to this function');
   }
   const directory = path.join(...directoryParts);
-  const directoryExists = await fs.exists(directory);
-  if (!directoryExists) {
-    throw new Error(`Directory: ${directory} does not exist`);
+  try {
```

Comment:
This try should be scoped as narrowly as possible (ideally, just to the rmdir part)